tests/end_to_end/test_osquery_integration/data/playbooks/configuration.yaml

```diff
@@ -0,0 +1,62 @@
+- name: Test configuration
+  hosts: wazuh-manager
+  become: true
+  tasks:
+
+    - name: Check if Osquery is installed or not on CentOS
+      command: systemctl status osqueryd --no-pager
+      register: osquery_check
+      when: ansible_facts['distribution'] == "CentOS"
+      ignore_errors: true
+
+    - name: Install Osquery on CentOS
+      shell: |
+        curl -LO https://pkg.osquery.io/rpm/osquery-4.5.1-1.linux.x86_64.rpm
+        rpm -ivh ./osquery-4.5.1-1.linux.x86_64.rpm
+      register: installation
+      when: (ansible_facts['distribution'] == "CentOS" and osquery_check.rc != 0 and osquery_check.rc != 3)
+
+    - name: Configure Osquery
+      copy:
+        src: "{{ configuration_file }}"
+        dest: /etc/osquery/osquery.conf
+        owner: root
+        group: root
+        mode: '0644'
+        backup: true
+      when: (osquery_check.rc == 3 or osquery_check.rc == 0 or installation.rc == 0)
+
+    - name: Configure the Osquery module
+      blockinfile:
+        path: /var/ossec/etc/ossec.conf
+        marker: <!-- {mark} ANSIBLE MANAGED BLOCK -->
+        insertbefore: ^</ossec_config>
+        block: |
+          <wodle name="osquery">
+          <disabled>no</disabled>
+          <run_daemon>yes</run_daemon>
+          <bin_path>/usr/bin</bin_path>
+          <log_path>/var/log/osquery/osqueryd.results.log</log_path>
+          <config_path>/etc/osquery/osquery.conf</config_path>
+          <add_labels>no</add_labels>
+          </wodle>
+
+    - name: Start Osquery
+      shell: systemctl start osqueryd
+      when: (osquery_check.rc == 3 or osquery_check.rc == 0)
+
+    - name: Restart the manager
+      shell: systemctl restart wazuh-manager
+
+    - name: Check if stress is installed or not on CentOS
+      command: rpm -q stress
+      register: stress_check
+      when: ansible_facts['distribution'] == "CentOS"
+      ignore_errors: true
+
```

Comment:
Put a comment indicating why it is necessary to ignore the errors in this task

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/3012/commits/6cf630d445793f9c5c3abc8afc1c9f24d1554647